R/utils-get_code_dependency.R

```diff
@@ -56,7 +56,7 @@ get_code_dependency <- function(code, names, check_names = TRUE) {
 
```

Comment:
line 55 loops by dataname - which means that line-idx are grouped by dataname. `sort` few lines later fixes this